target/linux/ramips/mt7620/config-5.4

```diff
@@ -206,6 +154,7 @@ CONFIG_PHY_RALINK_USB=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_RT2880=y
 # CONFIG_PINCTRL_SINGLE is not set
+CONFIG_PLUGIN_HOSTCC="g++"
```

Comment:
I don't think this should be added (applies to all six files).

Reply:
Is my host system somehow adding it? I only ran make kernel_oldconfig. I'll investigate.

Reply:
Do these need to be fixed too?
```
target/linux/apm821xx/config-5.4:CONFIG_PLUGIN_HOSTCC="g++"
target/linux/oxnas/config-5.10:CONFIG_PLUGIN_HOSTCC="g++"
target/linux/oxnas/config-5.4:CONFIG_PLUGIN_HOSTCC="g++"
target/linux/rockchip/armv8/config-5.4:CONFIG_PLUGIN_HOSTCC="g++"
target/linux/zynq/config-5.4:CONFIG_PLUGIN_HOSTCC="g++"
```

Reply:
Fixed in latest push. Submitting a separate patch on the ML suggesting a fix for PLUGIN_HOSTCC.